css/index.css

```diff
@@ -143,4 +178,49 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
+    text-align: center;
+    margin-left: 25%;
+}
+
+#about {
+    text-align: center;
+}
+
+
+#flex  {
+    width: 50%;
+}
+
```

Comment:
For the navigation, you also want to remove the text decorations so the underlines do not show up.